README.md

```diff
@@ -2,7 +2,7 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/portworx/ark-plugin)](https://goreportcard.com/report/portworx/ark-plugin)
 
 
-# ark-plugin
-Portworx plugin for Heptio Ark
+# velero-plugin
+Portworx plugin for Heptio Velero (formerly known as Ark)
```

Comment:
the project is now just "Velero", not "Heptio Velero"

Reply:
Updated